imports/plugins/custom/payments-paystack/lib/api/paystack.js

```diff
@@ -0,0 +1,36 @@
+import { Meteor } from "meteor/meteor";
+import { Packages } from "/lib/collections";
+
+export const Paystack = {
+  accountOptions: function () {
+    const settings = Packages.findOne({
+      name: "paystack-paymentmethod" // reaction-paymentmethod
+    }).settings;
+    if (!settings.secretKey && !settings.publicKey) {
+      throw new Meteor.Error("403", "Invalid Credentials");
+    }
+    return { secretKey: settings.secretKey, publicKey: settings.publicKey };
+  },
+
+  authorize: function (cardInfo, paymentInfo, callback) {
+    Meteor.call("paystackSubmit", "authorize", cardInfo, paymentInfo, callback);
+  },
+
+  verify: (ref, secretKey, fn) => {
```

Comment:
Hey Steve, please could you clarify where you got ref and secretKey from? I can't seem to find any reference to it in this PR

Reply:
Go fuck yourself!